alexandria/core/models.py

```diff
@@ -289,7 +289,13 @@ def get_webdav_url(self, username, group, host="localhost:8000"):
         payload = (username, group, str(self.pk))
         token = Token(key, path, payload=payload)
 
-        return f"{settings.ALEXANDRIA_MANABI_DAV_SCHEME}://{host}{settings.ALEXANDRIA_MANABI_DAV_URL_PATH}/{token.as_url()}"
+        handler = settings.ALEXANDRIA_MANABI_DAV_URI_SCHEMES.get(
+            self.mime_type, "ms-word:ofe|u|"
```

Comment:
why not no handler for undefined mime types?

Reply:
Fair enough, we could trigger an [`ImproperlyConfigured`](https://docs.djangoproject.com/en/5.0/ref/exceptions/#improperlyconfigured) exception if no handler was defined.

Reply:
Yeah sounds like a good idea :+1: instead of silently returning nothing

Reply:
Done